tests/unittest/evaluate/test_image_generation_eval_cli.py

```
@@ -0,0 +1,265 @@
+    import tensorrt_llm.evaluate.visual_gen.generators as generators
+    import tensorrt_llm.visual_gen as visual_gen
+
+    closed = []
+
+    class FakeVisualGenArgs:
+        @classmethod
+        def from_dict(cls, config):
+            return config
+
+    class FakeParams:
+        model_fields = {"seed": object()}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)